src/mfpbench/jahs/benchmark.py

```diff
@@ -77,7 +77,7 @@ class JAHSResult(Result[JAHSConfig, int]):
 
 class JAHSBenchmark(Benchmark[JAHSConfig, JAHSResult, int], ABC):
     JAHS_FIDELITY_NAME: ClassVar[str] = "epoch"
-    JAHS_FIDELITY_RANGE: ClassVar[tuple[int, int, int]] = (3, 200, 1)
+    JAHS_FIDELITY_RANGE: ClassVar[tuple[int, int, int]] = (1, 200, 1)
```

Comment:
This was previously the case but we had to make some changes for NePS related stuff. I'm happy this is set back to this :) Thank you!